primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -124,3 +124,75 @@ pub const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce = 1024;
 
 /// Maximal number of unconfirmed messages at inbound lane for Cumulus-based parachains.
 pub const MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX: MessageNonce = 4096;
+
+/// Module with rewarding bridge signed extension support
+pub mod rewarding_bridge_signed_extension {
```

Comment:
I'd remove all signing extensions from `bp-polkadot-core`, because we are no longer submit transactions to polkadot-like chains. And move proper struct and stuff to this crate.

Also I've just found that there are two signed extensions there - `BridgeSignedExtension` and `DefaultSignedExtension`. And it seems to me that `DefaultSignedExtension` may be safely removed and `BridgeSignedExtension` may be safely moved here.

It might be done in a separate PR though. I just find this `mod` and existing structs a bit confusing :)

Reply:
`DefaultSignedExtension` is used by chain-rialto-parachain now 
and `BridgeSignedExtension` I thought that it is like generic example, so I didnt want to touch it.

`BridgeSignedExtension` / `DefaultSignedExtension` both are types and implement `trait PolkadotSignedExtension`,
so I was not able to use `PolkadotSignedExtension` outside, so thats why I created this `mod`.

if you want to move this stuff out of `bp-polkadot-core`, I just agree and I can do it easily done now here,

but lets first merge Serban's PR: https://github.com/paritytech/parity-bridges-common/pull/1895
because `mod rewarding_bridge_signed_extension` as it is will be used for Kusama/Polkadot,
but for BridgeHubRococo/Wococo I probably will have two instance of `RefundBridgedParachainMessages`, so signed_extensions extra would have two `(), ()`, right?

Reply:
I've submitted https://github.com/paritytech/parity-bridges-common/issues/1896

Reply:
yeah, friday I just added one `RefundBridgedParachainMessages` to the runtime and I didnt touch relayer side where was `BridgeSignedExtension` (with BridgeRejectObsoleteHeadersAndMessages at the end) and the reward was registered, so I was also thinking that it is about some encoding stuff with `_` vs `()` vs `(), ()` 

I just created copies to the `runtime_wrapper.rs` with multiple `RefundBridgedParachainMessages` only for client-bridge-hub-rococo/wococo + TODO when to remove this in future